### DISCUSSION TITLE
Could you export `useRecentTransactions()` so we can import it and use it?

### DISCUSSION THREAD

**Discussion by smakosh** [rainbow-me/rainbowkit#868] [Ideas]
Similarly to what you've done with the modal hooks.

**Comment by shrugs** [comment]
duplicate: https://github.com/rainbow-me/rainbowkit/discussions/373

**Reply by smakosh** [reply]
Looks like I bumped it up there too lol, closing this for now (looks like I can't close it or delete it)
